tools/simple/grasshopper.c

```diff
@@ -22,6 +22,112 @@ static void generate_mul_tbl ();
 static void generate_coef_tbl ();
 static void generate_LS_tbl ();
 
+void copy_block(Block* dst, const Block* src) {
```

Comment:
I am not sure it's better than just call for memcpy cause this function may be non-inlined.  Unfortunately, inline creates a lot of problem with vectorization.

Anyway, you are free to add `__attribute__((always_inline))`.